jersey-tck/pom.xml

```diff
@@ -106,6 +106,11 @@
             <scope>test</scope>
         </dependency> 
 
+        <dependency>
```

Comment:
Is this plugin required in this module?

Reply:
Yes, the sigtest-maven-plugin built using https://github.com/jtulach/netbeans-apitest is the replacement of both sigtest.jar & sigtestdev.jar from jakartaee-tck. The same is used for both generating signature file and running the same.